packages/api/src/did-api.ts

```diff
@@ -54,54 +17,20 @@ export class DidApi {
     this.connectedDid = options.connectedDid;
   }
 
-  // constructor(options: DidApiOptions) {
-  //   const { didMethodApis, cache } = options;
-
-  //   this.didResolver = new DidResolver({ methodResolvers: options.didMethodApis, cache });
-
-  //   for (let methodApi of didMethodApis) {
-  //     this.methodCreatorMap.set(methodApi.methodName, methodApi);
-  //   }
-  // }
-
-  // /**
-  //  * Creates a DID of the method provided
-  //  * @param method - the method of DID to create
-  //  * @param options - method-specific options
-  //  * @returns the created DID
-  //  */
-  // create<M extends keyof CreateMethodOptions>(method: M, options?: CreateOptions<M>): Promise<DidState> {
-  //   const didMethodCreator = this.methodCreatorMap.get(method);
-  //   if (!didMethodCreator) {
-  //     throw new Error(`no creator available for ${method}`);
-  //   }
-
-  //   return didMethodCreator.create(options);
-  // }
-
-  // /**
-  //  * Resolves the provided DID
-  //  * @param did - the did to resolve
-  //  * @see {@link https://www.w3.org/TR/did-core/#did-resolution | DID Resolution}
-  //  * @returns DID Resolution Result
-  //  */
-  // resolve(did: string): Promise<DidResolutionResult> {
-  //   return this.didResolver.resolve(did);
-  // }
-
-  // /**
-  //  * can be used to add different did method resolvers
-  //  * @param _resolver
-  //  */
-  // addMethodResolver(_resolver: DidMethodResolver) {
-  //   throw new Error('not yet implemented');
-  // }
-
-  // /**
-  //  * can be used to add differed did method creators
-  //  * @param _creator
-  //  */
-  // addMethodCreator(_creator: DidMethodCreator) {
-  //   throw new Error('not yet implemented');
-  // }
+  /**
+   * Resolves a DID to a DID Resolution Result.
+   *
+   * @param didUrl - The DID or DID URL to resolve.
+   * @returns A promise that resolves to the DID Resolution Result.
+   */
+  async resolve(didUrl: string, resolutionOptions?: DidResolutionOptions): Promise<DidResolutionResult> {
```

Comment:
Will create be added eventually?

Reply:
Yes, absolutely.  Main goal of this PR was to get the plumbing setup to restore usage of `web5.did.*` since Angie has requested for some time that it be re-introduced.